src/app/shared/selectors/entry-file-selector.ts

```diff
@@ -42,6 +42,7 @@ export abstract class EntryFileSelector implements OnDestroy {
   public downloadFilePath: string;
   public customDownloadHREF: SafeUrl;
   public customDownloadPath: string;
+  public loading = false;
```

Comment:
Should be set to true initially.  Unlike Dockerfile, this time it is noticeable with the workflow descriptor file.  Appears to have no progress bar until after the first GA4GH endpoint or so is hit.  Then the progress bar shows up.  Easily replicate by blocking the first /files endpoint on Chrome and then reloading.  However...changing this will require sticking in more `loading = false` into more places.

Reply:
Edit: Oops, the blank screen on error is easily solved by using !content on the alert

Reply:
I discussed this in my commit message, which I realize now is not obviously visible to PR reviewers: [here](https://github.com/dockstore/dockstore-ui2/pull/502/commits/359085d13e012519ee2cee22f1c338a2d93ca68c)

Reply:
![image](https://user-images.githubusercontent.com/24548904/50235960-323f7a80-0387-11e9-9cd9-e394420fb943.png)

When the files endpoint has failed